packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -118,6 +118,17 @@ class ProjectUpdate(InputSchema):
     quality: dict[str, Any] = FieldNotRequired()
 
 
+class ProjectPatch(InputSchema):
+    name: str = Field(None)
```

Comment:
for all str identifiers, specially in the API schemas, use `IDStr` or something inherited from its